script/training_db_sync.rb

```diff
@@ -0,0 +1,142 @@
+# Script to update the 'training/pentest' environment's DB
+# from a copy taken from 'prod'.
+#
+# Run with: `bin/rails runner script/training_db_sync.rb`
+
+class TrainingDbSync
+  attr_reader :source, :destination, :data_filename, :pw_reset_script
+
+  ORGANISATION_NAME = "beis-report-official-development-assistance"
+
+  def initialize
+    @source = :prod
+    @destination = :pentest # this is the old name for 'training'
+    @data_filename = "/tmp/#{source}-dump-#{timestamp}.dump"
+    @pw_reset_script = "/tmp/pw_reset_script.rb"
+  end
+
+  def call
+    print_plan
+    capture_data_from_source
+    copy_data_to_destination
+    load_source_data_to_destination
+    force_password_reset_for_users
+    remove_temp_files
+  end
+
+  private
+
+  def authenticate_to(env)
+    cmd = "cf login -o #{ORGANISATION_NAME} -s #{env} -u #{cf_user} -p #{cf_password}"
+    CmdRunner.run(cmd)
+  end
+
+  def timestamp
+    DateTime.now.iso8601
+  end
+
+  def source_service
+    "beis-roda-#{source}-postgres"
+  end
+
+  def destination_service
+    "beis-roda-#{destination}-postgres"
+  end
+
+  def print_plan
+    plan =
+      "The *#{destination}* db will be dropped and replaced by a copy from *#{source}*. " \
+      "Users will need to reset their passwords on their next login to *#{destination}*."
```

Comment:
consider adding: "and they will need to re-register their mobile phone" or however the 2FA is impacted.